app/src/main/res/layout/layout_error.xml

```diff
@@ -20,7 +20,7 @@
         android:layout_height="wrap_content"
         android:layout_below="@+id/iv_status"
         android:layout_centerHorizontal="true"
-        android:layout_centerVertical="true"
+        android:padding="10dp"
```

Comment:
@sashank27 Why you are using padding here instead of `android:layout_centerVertical="true"`

Reply:
Actually, `android:layout_centerVertical="true"` causes the overlapping. So it has to be removed. And I added the padding so as to make them distinct